### PR TITLE
Add: Splash screen

### PR DESCRIPTION
**Pull request by jpirazusta**
Splash screen for iOS and Android.
Screenshots: 
Simulator Screen Shot - iPhone 11 - 2020-05-11 at 12 43 27
Screenshot_1589223536

### CODE REVIEW

File: android/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -3,6 +3,21 @@
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
+        <item name="android:textColor">#000000</item>
+
```
Comment (ManuViola77) [inline android/app/src/main/res/values/styles.xml]: this color should also be declared in the colors.xml

**Review by ManuViola77 — APPROVED** [review]
Thanks for teaching me how to do a splash screen! 👏  Nice job 😄

**Review by elizabethlofredo — APPROVED** [review]
💪

**Review by acollazomayer — APPROVED** [review]
Great Job

**Pull request merged**